lib/membrane_flac_plugin/membrane_flac_parser.ex

```diff
@@ -47,15 +56,37 @@ defmodule Membrane.FLAC.Parser do
     {[], state}
   end
 
+  defp set_current_pts(%{generate_best_effort_timestamps?: true} = state, _input_pts) do
+    state
+  end
```

Comment:
It is misleading, that function `set_current_pts` does nothing, if `state.generate_best_effort_timestamps? == true`. I guess that in this case `current_pts` will be set in `set_pts_audio_buffers`, but it is not clean. I recommend you renaming this function to something more relevant or changing it to simple `if` statement in `handle_buffer` (I guess it is used one time only there)